api/proto/teleport/legacy/types/types.proto

```diff
@@ -5337,6 +5339,23 @@ message PluginPagerDutySettings {
   string api_endpoint = 2;
 }
 
+message PluginJiraSettings {
+  option (gogoproto.equal) = true;
+
+  // ServerURL is the address of the target JIRA Server instance.
+  string server_url = 1;
+
+  // Project is the key of the Jira project that will receive
+  // notifications and issues from the plugin.
+  string project = 2;
```

Comment:
Nit: Is this a "project ID" or "project name" or something else? Maybe rename for clarity?